### PR TITLE
added vsc code snippets

### PR DESCRIPTION
**Pull request by xFuture603**
added some code snippets for Visual Studio Code.

For example: type "service-apply" and get the following code block back:
image

It was still unclear if this would fit into the repo, so I'm looking forward to hearing your thoughts about this. If it doesn't fit here I can create an extra repo and link to this one for example.

Instruction on how to use those snippets:
- open VSC
- File -> Preferences -> User Snippets
- add a new one for YAML context
- paste content from yaml.json in the new file
- save the file

### CODE REVIEW

**Comment by rndmh3ro** [comment]
For me, this fits into this collection.

We should however not include these into the built collection. They can be excluded via the galaxy.yml: https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html#ignoring-files-and-folders-collections

Also this should be added to the README. :)

**Comment by schurzi** [comment]
I also like this idea. Only the name `vsc-code`seems a bit redundant ;)

**Comment by xFuture603** [comment]
Hey @rndmh3ro and @schurzi.

Could you look through my latest changes again and see if everything fits now?

File: vsc-snippets/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+> Code snippets that you can import to your Visual Studio Code installation
+
+## Instruction on how to use those snippets:
```
Comment (rndmh3ro) [inline vsc-snippets/README.md:3]: This only tells me how to install the snippets. I don't know how to use snippets - is this straight-forward?

Do you want to maybe add an example or something?

Comment (xFuture603) [reply vsc-snippets/README.md]: Thanks for your suggestion. I have added a code example to the README.md

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**